package.json

```diff
@@ -109,14 +110,14 @@
     "babel-plugin-polyfill-corejs2": "^0.3.1",
     "cypress": "^10.3.1",
     "cypress-intellij-reporter": "^0.0.7",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
This package is to manage the formatting rules to avoid conflicts with prettier, I suppose. Is it not required to install eslint package separately, if we install this?